### ISSUE TITLE
Train/Val Split in DAIR‑V2X‑I

### ISSUE THREAD

**Issue by sucreil97** [ADLab-AutoDrive/BEVHeight#60]
May I ask whether you further split a validation set from the original training set of DAIR‑V2X‑I, and used the original validation set as the test set?
